src/main/java/nl/fuchsia/exceptionhandlers/ExceptionHandlers.java

```diff
@@ -6,9 +6,18 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
 
+/**
+ * Door onderstaande annotatie vangt deze classe posts op doe niet door de validatie komen.
+ */
```

Comment:
Is dit JavaDoc (bedoeld voor andere developers) of meer een notitie voor jezelf?

Reply:
Daarnaast is het bedoeld voor Excepties in het algemeen, niet specifiek voor validatie excepties. Jij gebruikt het op die manier.

Reply:
JavaDoc was bedoeld om voor Tjitte een korte spcifieke commentaar te geven. We hebben hem nu toch specifiek nodig vanwege om de vallidatie op te vangen zo dicht mogelijk bij de opvoer.